embedchain/embedchain.py

```diff
@@ -112,11 +112,9 @@ def load_and_embed(self, loader, chunker, src):
             documents, metadatas = zip(*data_dict.values())
 
         chunks_before_addition = self.count()
-
         self.collection.add(documents=documents, metadatas=list(metadatas), ids=ids)
         print(
             f"Successfully saved {src}. New chunks count: {self.count() - chunks_before_addition}"  # noqa:E501
-        )
```

Comment:
@ahnedeee this is the issue. Please add it back and it will work.

You can test your changes by running `make test` on your branch.